db/db.go

```diff
@@ -75,13 +76,12 @@ func connectToDatabase(connectionString string) (*sql.DB, error) {
 
 // Creates the connection string using Username, Password, hostname, and port
 func formConnectionString(Name string) string {
-	connectionString := fmt.Sprintf("%s:%s@tcp(127.0.0.1:%s)/", conf.Username, conf.Password, conf.Port)
+	connectionString := fmt.Sprintf("%s:%s@tcp(127.0.0.1:%s)/%s", conf.Username, conf.Password, conf.Port, Name)
```

Comment:
I believe "Name" should be lowercase here since its a local variable - and maybe change it to dbName for consistency?